backend/src/main/java/com/katelocate/menugenerator/recipe/RecipeRepository.java

```diff
@@ -27,14 +27,14 @@ public List<Recipe> findAll() {
     }
 
     public Optional<Recipe> findById(Integer id) {
-        return jdbcClient.sql("SELECT id, title, recipeType, body FROM Recipe WHERE id = :id")
+        return jdbcClient.sql("SELECT id, title, recipeType, ingredients, instructions FROM Recipe WHERE id = :id")
                 .param("id", id)
                 .query(Recipe.class)
                 .optional();
     }
 
     public void create(Recipe recipe) {
-        var updated = jdbcClient.sql("INSERT INTO Recipe (id, title, recipeType, ingredients, instructions) VALUES (?,?,?,?)")
+        var updated = jdbcClient.sql("INSERT INTO Recipe (id, title, recipeType, ingredients, instructions) VALUES (?,?,?,?,?)")
```

Comment:
nit: 9f477d50ef367e0a967be6a792bad412a932cce8 can be fixed up with a00d544580f8feadd22dc56bbae981444cb7225f